src/Admin/BaseMediaAdmin.php

```diff
@@ -75,7 +75,7 @@ public function getPersistentParameters()
         }
 
         $filter = $this->getRequest()->get('filter');
-        if ($filter && array_key_exists('context', $this->getRequest()->get('filter'))) {
+        if ($filter && \array_key_exists('context', $this->getRequest()->get('filter'))) {
```

Comment:
hmm, this change is already done in https://github.com/sonata-project/SonataMediaBundle/pull/1556 ... 🤔 and should not occur here if you rebased correctly.

Can you try to revert all the:
```diff
- array_key_exists(....
+ \array_key_exists(...
```

Reply:
They have been added by flint so I guess build will fail if I revert... right?
https://github.com/jo66/SonataMediaBundle/pull/2/files

Reply:
Maybe I should close this PR and open new one?

Reply:
@OskarStark I tried to revert but flint seems not happy